### ISSUE TITLE
Remote Socket Connection over TLS

### ISSUE THREAD

**Issue by mwolter805** [fmartinou/whats-up-docker#35]
Hello,
Would be great if you could implement remote socket connections over TLS as unencrypted connections leave root vulnerable. 

Thank you

**Comment by fmartinou** [comment]
Hi @mwolter805 ,
I have a first version supporting TLS
You can give it a try by running the latest `fmartinou/whats-up-docker:develop` version.

The documentation is not yet published but [you can find the draft here](https://github.com/fmartinou/whats-up-docker/tree/develop/docs/watchers)

Simple example of the new env vars to configure TLS:

```
WUD_WATCHER_MYREMOTEHOST_HOST="myremotehost"
WUD_WATCHER_MYREMOTEHOST_PORT="2376"
WUD_WATCHER_MYREMOTEHOST_CAFILE="/certs/ca.pem"
WUD_WATCHER_MYREMOTEHOST_CERTFILE="/certs/cert.pem"
WUD_WATCHER_MYREMOTEHOST_KEYFILE="/certs/key.pem"
```

Thanks for your feedback!

*N.B. The version is non backward compatible => Please remove /store/wud.json before upgrading*

**Comment by fmartinou** [comment]
It's now available in version `4.0.0`.
